src/components/KeywordGrab.js

```diff
@@ -83,34 +64,63 @@ const KeywordGrab = ({setGlobalState}) => {
                 return
             }
 
-            //https://suggestaurantapp-3sgrjmlphq-uc.a.run.app/
-            fetch("http://localhost:5000/data", {
-                method:"POST",
-                cache: "no-cache",
-                headers:{
-                    "content_type":"application/json",
-                    'Access-Control-Allow-Origin':'*'
-                },
-                body:JSON.stringify(
-                        jsonData
-                    )
-                }
-            ).then(response => {
-                return response.json();
-            })
-            .then(json => {
-                setCookie("businesslist", json, { path: '/' });
-                setGlobalState({'businesslist': json});
+            setCookie('keywords', keywordRef.current.value, { path: '/' });
+
+            const check = user.isAnonymous ? null : getFilters(user.uid).then(val => {
```

Comment:
the fetch block is coming back as a change. If merged, this will be added to keywords again. Looks like you need to update from main and put at least some of these changes in the UserWaiting.js file that's in Main.